src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecLoaderExtensions.cs

```diff
@@ -16,13 +16,45 @@
 
 using System;
 using System.IO;
+using System.Linq;
+using System.Reflection;
 using System.Text;
+using Nethermind.Config;
 using Nethermind.Logging;
 
 namespace Nethermind.Specs.ChainSpecStyle
 {
     public static class ChainSpecLoaderExtensions
     {
+
+        public static ChainSpec LoadFromEmbeddedOrFile(this IChainSpecLoader chainSpecLoader, string fileName, ILogger logger)
```

Comment:
I think from the English perspective, better to rename to `LoadEmbeddedOrFromFile`.